topics/assembly/tutorials/unicycler-assembly/tutorial.md

```diff
@@ -0,0 +1,268 @@
+
+### Annotation
+
+For annotation we are using [Prokka](http://www.vicbioinformatics.com/software.prokka.shtml) (also see [Seeman:2014](https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btu153)). It scans the assembly generated with Unicycler with a set of feature prediction tools and compiles a list of genome annotation. It predicts the following features (Table from [Seeman:2014](https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btu153)):
+
+| Feature | Tool used by Prokka |
+|---------|----------------------|
+| Protein-coding sequences (CDS) | [Prodigal](https://github.com/hyattpd/Prodigal) |
+| Ribosomal RNA genes | [RNAmmer](http://www.cbs.dtu.dk/cgi-bin/nph-runsafe?man=rnammer) |
+| Transfer RNA genes | [Aragorn](https://www.ncbi.nlm.nih.gov/pubmed/14704338) |
+| Signal leader peptides | [SignalP](https://www.ncbi.nlm.nih.gov/pubmed/21959131) |
+| Non-coding RNA genes | [Infernal](http://eddylab.org/infernal/) | 
+
+Prokka predicts protein-coding regions using a two step process. It first identifies coordinates of putative genes using [Prodigal](https://github.com/hyattpd/Prodigal) and then compares the gene sequence against databases of known sequences at protein level using [Blast+](https://www.ncbi.nlm.nih.gov/books/NBK279690/) and [HMMer](http://hmmer.org/).
+
+# Let's try it
+
+
+## Load data and assess quality
+
+In this example we will use a downsampled version of *E. coli* C-1 Illumina and ONT sequencing data. These include 3 files: forward and reverse reads for Illumina, and Long read file produced by ONT. All data are in [fastq](https://en.wikipedia.org/wiki/FASTQ_format) format. 
+
+### Load data into History
+
+To load data into your Galaxy instance log in into Galaxy and create new history (if you are new to Galaxy see [Galaxy 101 tutorial](/topics/introduction/tutorials/galaxy-intro-101/tutorial.html) first). 
+
+Click **Get data** icon as shown below (see [these slides](/topics/introduction/tutorials/galaxy-intro-get-data/slides.html) for an introduction on how to load data into Galaxy):
```

Comment:
To make the hands-on sections more visible during workshops and self-training, we decided to put them into boxes like:
![screen shot 2017-10-05 at 19 04 03](https://user-images.githubusercontent.com/1842467/31240915-798b0f9c-aa02-11e7-9bea-00cdd3d92080.png)

Can you add the hands-on sections of your tutorial into such boxes? Something that looks like:

```
> ### {% icon hands_on %} Hands-on: Align sequences
>
> 1. Import the `silva.v4.fasta` file in your history
> 2. **Align.seqs** {% icon tool %} with the following parameters
>   - "fasta" to the `good.fasta` output from `Screen.seqs`
>   - "reference" to the `silva.v4.fasta` reference file
>   - "flip" to `Yes`
>
>    This step may take a few minutes, please be patient.
>
> 3. **Summary.seqs** {% icon tool %} with the following parameters
>   - "fasta" parameter to the aligned output from `Align.seqs`
>   - "count" parameter to count_table output from `Screen.seqs`
>
{: .hands_on}
```